src/view/com/composer/Prompt.tsx

```diff
@@ -3,13 +3,15 @@ import {StyleSheet, TouchableOpacity} from 'react-native'
 import {UserAvatar} from '../util/UserAvatar'
 import {Text} from '../util/text/Text'
 import {usePalette} from 'lib/hooks/usePalette'
-import {useStores} from 'state/index'
 import {useWebMediaQueries} from 'lib/hooks/useWebMediaQueries'
 import {Trans, msg} from '@lingui/macro'
 import {useLingui} from '@lingui/react'
+import {useSession} from '#/state/session'
+import {useProfileQuery} from '#/state/queries/profile'
 
 export function ComposePrompt({onPressCompose}: {onPressCompose: () => void}) {
-  const store = useStores()
+  const {currentAccount} = useSession()
+  const {data: profile} = useProfileQuery({did: currentAccount!.did})
```

Comment:
This is actually a place where we want to handle the possibility of no session

Reply:
Yep yep, the `useProfileQuery` has the `enabled` check, and `UserAvatar` falls back to `DefaultAvatar` in the case of undefined. That work or were you thinking of something else?

Reply:
Tbh I'd rather not rely on those enable checks much if we can help it, but we'll need to for the time being

Reply:
Wont `currentAccount!.did` reference error if `currentAccount` is undefined?